### PR TITLE
Issue/material stats widget configuration

### PR DESCRIPTION
**Pull request by khaykov**
This PR moves Stats widget configuration to the material theme and adds dark mode support.

We are using Subtitle 1 text appearance for the previously bold labels.

comparison_st

Only one designer and one engineer are required to review this PR, but any is welcome to join.

To test:
- Make sure that configuration activities for all the widgets look ok in both light and dark themes.

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by khaykov** [comment]
Ah, good catch, @develric ! I hope you don't mind that I updated the look of that widget in this PR. It's super old, so I had to change couple of things, but I tried no to get too involved :)

comparison_qp

Since widget screens are pretty simple, I think we can skip design review, for now, to speed up things a bit. We can address any potential issue during the final check.

File: WordPress/src/main/res/layout/quick_press_widget_configure_list_row.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="?android:selectableItemBackground"
+    android:orientation="vertical"
```
Comment (develric) [inline WordPress/src/main/res/layout/quick_press_widget_configure_list_row.xml]: super minor np: I guess this can be here from some copy/paste? wdyt?

Comment (khaykov) [reply WordPress/src/main/res/layout/quick_press_widget_configure_list_row.xml]: Yep, this not supposed to be there :)

**Comment by khaykov** [comment]
Thank you for review, @develric !

**Pull request merged**